README.md

```diff
@@ -1,40 +1,198 @@
-# Vault Integration Pack
-
-This pack is for Hashicorp Vault integrations
+# vault integration pack
```

Comment:
Could you also add yourself to the list of the pack Maintainers in the README and via Codeowners?

The format is the following per https://github.com/StackStorm-Exchange/exchange-incubator/issues/152

https://github.com/StackStorm-Exchange/stackstorm-jira#maintainers
https://github.com/StackStorm-Exchange/stackstorm-jira/blob/master/.github/CODEOWNERS

Helps keeping things more organized with the pack maintainers and clear ownership.

Reply:
Noticed there was already a PR for that: https://github.com/StackStorm-Exchange/stackstorm-vault/pull/24, - just merged that.

Could you add yourself to the list of pack maintainers in the README.md as well? (the email is optional)

Reply:
Added myself as a codeowner